backend/app/channels/game_channel.rb

```diff
@@ -0,0 +1,22 @@
+class GameChannel < ApplicationCable::Channel
+  def subscribed
+    stream_from "game_channel_#{current_user.id}"
+  end
+  def unsubscribed
+    Game.leave_game(current_user) if current_user.present?
+  end
+
+  def joinGame(data)
+    Game.join(current_user) if current_user.present?
+  end
+  def leaveGame(data)
+    Game.leave_game(current_user) if current_user.present?
```

Comment:
So why this happens, all games are left? What if I have two sessions and want to leave a game in one?

Reply:
Yes In this version i am not supporting multiple sessions